docs/jupiterone.md

```diff
@@ -91,18 +76,28 @@ https://github.com/JupiterOne/sdk/blob/master/docs/integrations/development.md
 
 The following entities are created:
 
-| Resources    | Entity `_type`     | Entity `_class`           |
-| ------------ | ------------------ | ------------------------- |
-| Organization | `tfe_organization` | `Account`, `Organization` |
-| User         | `tfe_user`         | `User`                    |
+| Resources       | Entity `_type`           | Entity `_class`           |
+| --------------- | ------------------------ | ------------------------- |
+| Account         | `tfe_account`            | `User`                    |
+| Entitlement Set | `tfe_entitlement_set`    | `Entity`                  |
+| Organization    | `tfe_organization`       | `Account`, `Organization` |
```

Comment:
Do we want to remove 'Account' from this entity's _class list if we're keeping the tfe_account entity?

Reply:
Same here!